types/chains.go

```diff
@@ -0,0 +1,54 @@
+package types
+
+import "github.com/attestantio/go-eth2-client/spec/phase0"
+
+// DomainType is a unique identifier for signatures, 2 identical pieces of data signed with different domains will result in different sigs
+type DomainType [4]byte
+
+// Chains are intended to separate different SSV chains. A chain can have many forks in it.
+const (
+	MainnetChain = 0x0
+	PrimusChain  = 0x1
+	ShifuChain   = 0x2
+	JatoChain    = 0x3
+)
+
+// DomainTypes represent specific forks for specific chains, messages are signed with the domain type making 2 messages from different domains incompatible
+var (
+	GenesisMainnet = DomainType{0x0, 0x0, MainnetChain, 0x0}
+	PrimusTestnet  = DomainType{0x0, 0x0, PrimusChain, 0x0}
+	ShifuTestnet   = DomainType{0x0, 0x0, ShifuChain, 0x0}
+	ShifuV2Testnet = DomainType{0x0, 0x0, ShifuChain, 0x1}
```

Comment:
do we need to have deprectated nets here?

Reply:
we can later

Reply:
what does the 2 first bytes stand for ?

Reply:
future TBD use (just have space for it)